pom.xml

```diff
@@ -125,6 +125,12 @@ under the License.
             <version>2.10.1-9.0</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-yarn_${scala.binary.version}</artifactId>
+            <version>${flink.version}</version>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
As `flink-yarn` module is not included in the default Flink distribution (users need to download an additional flink hadoop library), we should not depend on it.

Shall we put all these yarn related constant strings into a new class and state that the new class should be in sync with the `flink-yarn` module?

Reply:
Thanks for your review. I will remove this dependence and add a new class named YarnConfigOptions in package com.ververica.flink.table.gateway.config